pkg/virt-launcher/virtwrap/network/common_test.go

```diff
@@ -31,7 +31,31 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 )
 
+type ipTestsParams struct {
+	Cidr         string;
+	TooSmallCidr string;
+	Gw           string;
+	Vm           string;
+}
+
 var _ = Describe("Common Methods", func() {
+	testParams:=  struct {
```

Comment:
Each test has its own relevant ips, the ips are not reused. For me, it doesn't make sense to aggregate them.

Reply:
Most of the tests in this file related to IP addresses, even though its not reused it easier to maintain those tests this way in case of any change.
It easier to understand when you have all the addresses in one place.

Reply:
The struct does not seem helpful to me, it adds a lot of noise and contributes almost nothing.
You could use simple string constants with proper naming and reason it with: Give the various addresses representative names and encourage future tests in this context to use them.

I do tend to agree with Alona here, the reasoning in general for such a change is weak.
Said that, using constants instead of strings is always a better choice.

Reply:
Thanks for the feedback I totally understand your point,
my intent was to make this test easy to understand and make it more easy to maintain with concentrating all hard-coded addresses in one place.
I dropped this commit for now.